src/bikeracks/templates/base.html

```diff
@@ -83,11 +83,24 @@
                                   aria-haspopup="true" aria-expanded="false">
                                   Show Racks
                           </button>
-                            <div class="dropdown-menu">
+                            <ul class="dropdown-menu">
+                                  <li>
+                                    <input id="showApproved" name="showApproved"
+                                           class="checkboxes"
```

Comment:
Maybe this would be better as radio buttons? Having no checked boxes and both checked boxes be the same seems odd to me.

Reply:
How does this already work? It's not working for me 😆.

Reply:
Because of the click handlers you have hooked up to the divs. They don't care that it's a checkbox, they just wait for you to click and then do something. It just so happens that the behavior matched the checkbox state when I looked at it.

If you've deleted that stuff or done something else then it probably doesn't work anymore.